### PR TITLE
alias alternative for windows

### PR DESCRIPTION
**Pull request by rdparihar**
alias configuration for dos/windows based system using doskey

### CODE REVIEW

File: site/content/en/docs/start/_index.md [hunk 1]
```diff
@@ -525,6 +525,11 @@ You can also make your life easier by adding the following to your shell config:
 ```shell
 alias kubectl="minikube kubectl --"
 ```
+for windows :
```
Comment (spowelljr) [inline site/content/en/docs/start/_index.md:528]:
```suggestion
For Windows:
```

**Review by spowelljr — CHANGES_REQUESTED** [review]
Does this work for CMD and PowerShell?

Could you update the text to indicate what it works with.

Example:
`For Windows with CMD & PowerShell:`

File: site/content/en/docs/start/_index.md (hunk 1 above)
Comment (rdparihar) [reply site/content/en/docs/start/_index.md:528]: this works for Windows with CMD 

for windows with powershell 

function kubectl { minikube kubectl -- $args }

Comment (medyagh) [inline site/content/en/docs/start/_index.md:528]: Lets make this Automatic based on user's browser, same as download instructions that automatically goes

"https://minikube.sigs.k8s.io/docs/start/"

**Review by medyagh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**